addons/website/views/website_templates.xml

```diff
@@ -965,21 +969,48 @@
     </div>
 </template>
 
+<template id="400">
+    <t t-call="website.layout">
+        <div id="wrap">            
+            <t t-if="request.debug">
+                <div class="container">
+                    <h1 class="mt32">Oops! Something went wrong.</h1>
```

Comment:
Use mt-5 (or another size value with -) instead so it can be more easily customized (by themes or changing `$spacer` value). (everywhere you made a diff, but don't change in all Odoo)